charts/add-ons/prometheus/templates/prometheus.yaml

```diff
@@ -231,6 +231,34 @@ spec:
       securityContext:
         fsGroup: 65534
       containers:
+      {{- range .Values.sidecarContainers }}
```

Comment:
I was thinking why not let the user define the full container info in `values.yaml` and we just put that here without having to deal with each individual field. :thinking:

Reply:
I went back and forth on this a lot, tbh, and ended up coming down on the side of ensuring that the /data and config mounts were always there, but I think you're right: we can just put that in the example.